ontologies/vocabs.ttl

```diff
@@ -1138,8 +1118,10 @@ Tag for the Government industry sector"""^^xsd:string ;
 
 gist:_IndustrySector_government-Local
 	a gist:IndustrySector ;
-	rdfs:label "Government Local"^^xsd:string ;
-	skos:altLabel "Government-Local"^^xsd:string ;
+	skos:altLabel
+		"Government Local"^^xsd:string ,
+		"Government-Local"^^xsd:string
```

Comment:
Is this helpful?
Also, should there not be a stixTerm? I guess that is a different git issue?